dev-tools/globals.ts

```diff
@@ -538,7 +539,7 @@ interface Move extends Effect, MoveData {
 	zPowered?: boolean
 }
 
-type TemplateAbility = {0: string, 1?: string, H?: string, S?: string}
+type TemplateAbility = {0: string, 1?: string, H?: string, S?: string, [key: string]: string | undefined}
```

Comment:
Same thing here, not sure what code is causing you to make this change but adding `[key: string]: string` isnt correct, not sure about the undefined either but I wont comment on that until I know why its undefined. Also you would probably want `?` rather than adding `undefined` to the list.

Reply:
It looks as if the syntax would be `[key: string]?: string` but I don't think that works yet. Oh well, I thought it was a better solution than `@ts-ignore`...

Reply:
Not really because it makes every check for an ability a lot less strict and prone to errors, I think id rather mark one specific case we know is ok as such with `// @ts-ignore` than make abilities all over the sim prone to more errors.